### ISSUE TITLE
Team Drive Support

### ISSUE THREAD

**Issue by kinv** [l3uddz/drive_strm#15]
Hi, 

There seems to be an issue when using TD's in logs I see processing 460 changes, however only a small amount of strm files are pulled. 

```

2019-04-23 09:43:42,276 - INFO       - utils.google                                       - _process_changes                    - Processing 459 changes


2019-04-23 09:43:43,342 - INFO       - utils.google                                       - _process_changes                    - 0 added / 0 removed

```

With debug logging

```

2019-04-23 09:57:46,926 - INFO       - drive_strm                                         - run                                 - Starting STRM Server on 0.0.0.0:7294


2019-04-23 09:57:47,128 - DEBUG      - utils.google                                       - query                               - Request URL: https://www.googleapis.com/drive/v3/changes?pageToken=7143&pageSize=1000&includeRemoved=True&includeTeamDriveItems=True&supportsTeamDrives=True&fields=changes%28file%28md5Checksum%2CmimeType%2CmodifiedTime%2Cname%2Cparents%2CteamDriveId%2Ctrashed%29%2CfileId%2Cremoved%2CteamDrive%28id%2Cname%29%2CteamDriveId%29%2CnewStartPageToken%2CnextPageToken


2019-04-23 09:57:47,128 - DEBUG      - utils.google                                       - query                               - Request ARG: {'params': {'pageToken': '7143', 'pageSize': 1000, 'includeRemoved': True, 'includeTeamDriveItems': True, 'supportsTeamDrives': True, 'fields': 'changes(file(md5Checksum,mimeType,modifiedTime,name,parents,teamDriveId,trashed),fileId,removed,teamDrive(id,name),teamDriveId),newStartPageToken,nextPageToken'}}


2019-04-23 09:57:47,128 - DEBUG      - utils.google                                       - query                               - Response Status: 200 OK


2019-04-23 09:57:47,129 - INFO       - utils.google                                       - _process_changes                    - Processing 0 changes


2019-04-23 09:57:47,129 - INFO       - utils.google                                       - _process_changes                    - 0 added / 0 removed

```

**Comment by Kongledyret** [comment]
Did it actually generate streams for use? How did you setup it with teamdrives?

**Comment by kinv** [comment]
It generated a couple. But not much. Looks like its some sort of file permission issue with the teamdrive api this project uses.

there is an option for teamdrive true or false, set to true and put in the teamdrive names.

**Comment by l3uddz** [comment]
Resolved by commit: 4a05e1b
